### PR TITLE
NXP-27961: Replace xvfb with vnc server

### PR DESCRIPTION
**Pull request by doulba**

### CODE REVIEW

File: builder-java-firefox/Dockerfile [hunk 1]
```diff
@@ -15,8 +15,12 @@ ARG BASE_IMAGE
 
 FROM ${BASE_IMAGE}
 
-# Install Firefox
+# Install tigervnc-server and Firefox
 ARG FIREFOX_VERSION
+
+RUN yum install tigervnc-server xterm -y && \
+  yum clean all
+
```
Comment (troger) [inline builder-java-firefox/Dockerfile]: I would have kept the `ARG FIREFOX_VERSION` next to where we use it.
```
# Install tigervnc-server
RUN yum ...
...

# Install Firefox
ARG FIREFOX_VERSION
...
```

Comment (doulba) [reply builder-java-firefox/Dockerfile]: Hi,
changes have been applied

File: builder-java-firefox/Dockerfile [hunk 2]
```diff
@@ -15,6 +15,10 @@ ARG BASE_IMAGE
 
 FROM ${BASE_IMAGE}
 
+# Install tigervnc-server
+RUN yum install tigervnc-server xterm -y && \
```
Comment (ataillefer) [inline builder-java-firefox/Dockerfile]: do we need xterm? what for?

Comment (doulba) [reply builder-java-firefox/Dockerfile]: from the page describing how to fixe the random error. 
Pearhaps in our case we don't need xtem to be installed.

Comment (ataillefer) [reply builder-java-firefox/Dockerfile]: I don't know, I think it's worth understanding what it does, if it is required, etc.

**Comment by troger** [comment]
I would reword the commit to at least remove the "fix replace"...

```
NXP-27961: Replace xvfb with vnc server
```
Should be enough.

**Comment by doulba** [comment]
> I would reword the commit to at least remove the "fix replace"...
> 
> ```
> NXP-27961: Replace xvfb with vnc server
> ```
> 
> Should be enough.

the commit message have been updated.
